composer.json

```
@@ -16,32 +16,31 @@
   }
   ],
   "require" : {
-    "php" : ">=8.1",
+    "php" : ">=8.2",
```

Why is this upgraded? Is it just for symfony 7? In this case it may be possible to keep it to 8.1 as it is still [supported](https://www.php.net/supported-versions.php).

```suggestion
    "php" : ">=8.1",
```

---

the problem is [symfony/validator](https://packagist.org/packages/symfony/validator#v7.0.0) itself